### DISCUSSION TITLE
Configuration websocket

### DISCUSSION THREAD

**Discussion by lzm93** [nioc/xmpp-web#117] [Q&A] [answered]
Olá colegas, estou iniciando em openfire, e gostaria de uma ajuda de vocês. tenho meu servidor openfire e não consigo acesso websocket dele tem alguma documentação ou alguém pode me ajudar nessa parte?

**Comment by nioc** [comment]
English please 😉

**Comment by lzm93** [comment]
Hello colleagues, I'm starting out in OpenFire, and I would like some help from you. I have my openfire server and I can't access websocket from it. Is there any documentation or can anyone help me with this part?
I use openfire 4.8

**Reply by nioc** [reply, marked as answer]
I use Prosody which uses http ports 5280/5281 (ssl) for websocket as described in [their doc] (https://prosody.im/doc/modules/mod_websocket).

I can't guarantee that the app works with openfire but according to [their doc](https://www.igniterealtime.org/projects/openfire/plugins/1.2.1/websocket/readme.html) it's quite similar with port 7443 (ssl). 
Proxify `https://your.openfire.host:7443/ws/` in the docker image with the `XMPP_WS` env var (which will set the [Nginx conf](https://github.com/nioc/xmpp-web/blob/master/docs/docker/default.conf.template)).

With an Apache proxy, the protocol is `ws(s)` instead of `http(s)`.
